utils.ts

```diff
@@ -111,3 +113,16 @@ const USDGLO_POLYGON_CONTRACT_ADDRESS =
 export const getUSDCToUSDGLOUniswapDeeplink = (amount: number): string => {
   return `https://app.uniswap.org/#/swap?inputCurrency=${USDC_POLYGON_CONTRACT_ADDRESS}&outputCurrency=${USDGLO_POLYGON_CONTRACT_ADDRESS}&exactAmount=${amount}&exactField=input&chain=polygon`;
 };
+
+export const getBalance = async (address: string): Promise<BigNumber> => {
+  const provider = new ethers.providers.JsonRpcProvider(
+    process.env.POLYGON_RPC_URL
+  );
+  const abi = ["function balanceOf(address account) view returns (uint256)"];
+  const usdgloContract = new ethers.Contract(
+    USDGLO_POLYGON_CONTRACT_ADDRESS,
```

Comment:
You could use config.ts so it also works on testnet (and we dont have same contract address in N places)

Reply:
Nice!

Reply:
Updated but the code will always just default to the defaultChainId now which is polygonMumbai
Maybe this page needs a chain selector? cc @gglucass

Reply:
Yeah I think the chain selector is inevitable, despite me trying to hold off on it for so long :p

Maybe we do a subtle one like Uniswap has?
<img width="212" alt="image" src="https://github.com/Glo-Foundation/glo-wallet/assets/1226415/bd927235-1d11-4361-baaf-52cd8d477922">